net/openvpn/files/openvpn.init

```diff
@@ -144,6 +144,7 @@ openvpn_add_instance() {
 	local security="$4"
 	local up="$5"
 	local down="$6"
+	local client="$7"
```

Comment:
on some case we have to check the config file, to determine it is client or not.
use code:
```
local client=$(grep -q "mode.*server" "/var/etc/$conf" || echo 1)$7
```